spanner/MANIFEST.in

```diff
@@ -1,5 +1,4 @@
 include README.rst LICENSE
-include google/cloud/spanner_v1/gapic/transports/spanner.grpc.config
```

Comment:
Nope, this undoes #6928.  @crwilcox Do we need a `synth.py` change to prevent this?